File: Technology Fundamentals/02.Exercises - Intro and Basic Syntax/JavaScript/printAndSum.js
function solve(startNumber, endNumber){
    let range = [];
    let sum = 0;
    for(let i = startNumber; i <= endNumber; i++){
        range.push(i);
        sum +=i;
    }

    let output = "";
    for(let i = 0; i < range.length; i++){
        output += range[i] + " ";
    }

    console.log(output);
    console.log(`Sum: ${sum}`);
}
solve(5,10);
solve(50,60);

File: C# Fundamentals May 2019/02.Exercises - Intro and Basic Syntax/Java/src/PrintAndSum.java
import java.util.ArrayList;
import java.util.Scanner;

public class PrintAndSum {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int startNumber = Integer.parseInt(scanner.nextLine());
        int endNumber = Integer.parseInt(scanner.nextLine());

        int sum = 0;

        ArrayList<Integer> range = new ArrayList<Integer>();

        for (int i = startNumber; i <=endNumber; i++)
        {
            sum += i;
            range.add(i);
        }

        for(int i = 0; i < range.size(); i++){
            System.out.print(range.get(i) + " ");
        }
        System.out.println();
        System.out.println("Sum: " + sum);
    }
}
